### PR TITLE
Update to lastic-grant3.md

### PR DESCRIPTION
**Pull request by poppyseedDev**
#### Revised Proposal Updates

Following the completion of our milestone, we encountered several unforeseen changes to our original proposal. Here are the key updates:

- We have removed Pierina from the proposal.
- We have decided against using a local version of the Rococo chain for indexing. Initially, we considered indexing from a locally run Rococo chain, but this approach significantly complicated the verification process. Furthermore, with the Rococo coretime chain operating effectively, the need for a locally run indexer is eliminated. This was only considered a necessary measure when the Rococo coretime chain was not functioning properly.
- We have also decided against indexing the Rococo relay chain's information. This decision stems from realizing that the essential data can be effectively sourced from real-time data. Operating three indexers for the Rococo, Kusama, and Polkadot relay chains was deemed not financially viable when alternative methods are available.
- Due to these changes, the budget required for the first milestone has been reduced by $3,000. Conversely, we've expanded the scope for the second milestone by $3,000. Instead of producing a basic mock-up code, we are now enhancing the full functionality of the presented pages.
- Additional minor adjustments involve our data display approach. Initially, data obtained from Subscan did not meet our expectations in certain areas. To address this, we have supplemented it with data directly from the Polkadot JS API.

### CODE REVIEW

**Review by keeganquigley — APPROVED** [review]
Thanks @poppyseedDev by "grants" I think you mean "milestone" costs rather, but other than that LGTM.

**Comment by semuelle** [comment]
> - Due to these changes, the budget required for the first grant has been reduced by $3,000. Conversely, we've expanded the scope for the second grant by $3,000. Instead of producing a basic mock-up code, we are now enhancing the full functionality of the presented pages.

Just to clarify, this is referring to milestones 1 and 2, not grants.

**Review by Noc2 — APPROVED** [review]

File: applications/lastic-grant3.md [hunk 1]
```diff
@@ -232,63 +213,31 @@ Pierina's unique trajectory blends her expertise in health informatics with her
...
 | **0d.** | Article Publishing | Publication of an article elaborating on Lastic's current development. |
-| **1.** | Subscan Visualizations | Development of visualizations for Polkadot and Kusama. This includes auction history, a timer view of the auction process, a list of all connected parachains and parathreads, and detailed information on different PARA IDs. Includes steps 1a. - 1d. |
+| **1.** | Functional Visualizations | Development of visualizations for Polkadot and Kusama. This includes auction history, a timer view of the auction process, a list of all connected parachains and parathreads, and detailed information on different PARA IDs. Includes steps 1a. - 1d. |
 | **1a.** | Auction History | Includes visualizations and a list detailing the teams who won the slot over time, the duration of that slot, and other relevant information. |
-| **1b.** | Timer View | A visualization showing the current status and timeline of ongoing auctions. |
+| **1b.** | Timer View | A visualization showing the current status and timeline of ongoing of ongoing sales of coretime. |
```
Comment (PieWol) [inline applications/lastic-grant3.md:230]:
```suggestion
| **1b.** | Timer View | A visualization showing the current status and timeline of ongoing sales of coretime. |
```

**Review by PieWol — APPROVED** [review]
generally looks good to me.

**Comment by poppyseedDev** [comment]
> Thanks @poppyseedDev by "grants" I think you mean "milestone" costs rather, but other than that LGTM.

> > * Due to these changes, the budget required for the first grant has been reduced by $3,000. Conversely, we've expanded the scope for the second grant by $3,000. Instead of producing a basic mock-up code, we are now enhancing the full functionality of the presented pages.
> 
> Just to clarify, this is referring to milestones 1 and 2, not grants.

Yes correct, thank you for the clarification, I fixed the wording.

**Review by keeganquigley — APPROVED** [review]

**Review by Noc2 — APPROVED** [review]

**Pull request merged**